prereise/gather/demanddata/transportation_electrification/mileage.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+import pandas as pd
+from scipy.io import loadmat
+
+from prereise.gather.demanddata.transportation_electrification import const
+
+
+def get_model_year_dti(model_year: int) -> pd.DatetimeIndex:
+    return pd.date_range(
+        start=f"{model_year}-01-01", end=f"{model_year}-12-31", freq="D"
+    )
+
+
+def get_input_day(model_year_dti: pd.DatetimeIndex) -> np.ndarray:
+    """Determine whether each day of the model year is a weekend (1) or weekday (2)
+
+    :param pd.DatetimeIndex model_year_dti: a DatetimeIndex encompassing the model year.
+    :return: (*np.ndarray*) array of 1s and 2s indicating weekend/weekday designations for the model year.
+
+    """
+    return model_year_dti.dayofweek.isin(range(5)).astype(int) + 1
+
+
+def get_input_month(model_year_dti: pd.DatetimeIndex) -> np.ndarray(int):
+    """Determine month of each day
+
+    :param pd.DatetimeIndex model_year_dti: a DatetimeIndex encompassing the model year.
+    :return: (*np.ndarray*) -- stores the month the day is in for each day of the year.
+    """
+    return model_year_dti.month
+
+
+def get_data_month(data: pd.DataFrame) -> pd.Series:
+    """Get month value from data.
+
+    :param pd.DataFrame data: the data to get months from.
+    :return: (*pd.Series*) -- list of months.
+    """
+    return data["Date"].dt.month
+
+
+def get_data_day(data: pd.DataFrame) -> np.array(int):
+    """Get weekday/weekend designation value from data.
+
+    :param pd.DataFrame data: the data to get day of week from.
+    :return: (*np.array*) -- indicates weekend or weekday for every day.
+    """
+    return data["If Weekend"]
+
+
+def load_data(census_region: int, filepath: str = "nhts_census.mat") -> pd.DataFrame:
+    """Load the data at nhts_census.mat.
+
+    :param int census_region: the census region to load data from.
+    :param str filepath: the path to the matfile.
+    :raises ValueError: if the census division is not between 1 and 9, inclusive.
+    :return: (*pd.DataFrame*) -- the data loaded from nths_census.mat, with column
+        names added.
+    """
+    if not (1 <= census_region <= 9):
+        raise ValueError("census_region must be between 1 and 9 (inclusive).")
+
+    # Somehow load in the matfile
+    nhts_census = loadmat(filepath)
+    # Load the data for the desired region
+    raw_data = nhts_census[f"census_{census_region}_sorted"]
+    # Convert to data frame, adding column names
+    df = pd.DataFrame(raw_data, columns=const.nhts_census_column_names)
+
+    return df
+
+
+def remove_ldt(data: pd.DataFrame) -> pd.DataFrame:
+    """Remove light duty trucks from data loaded from nths_census.mat.
+
+    :param pd.DataFrame data: the data returned from load_data.
+    :return: (*pd.DataFrame*) -- the data loaded from load_data with all rows

```

Comment:
load_data --> :func:\`load_data\`

Reply:
Some more context: formatting the docstring this way results in Sphinx automatically creating a link to the referenced function within the documentation. See https://breakthrough-energy.github.io/docs/powersimdata.utility.html#powersimdata.utility.distance.angular_distance for an example.